### ISSUE TITLE
How can I train on a dataset without lidar2cam matrix?

### ISSUE THREAD

**Issue by Italian-PAO** [saic-vul/imvoxelnet#59]
Hi filaPro,
Thank you for your brilliant work in 3D detection.
I'm trying to train imvoxelnet on my own dataset which only has world2cam matrix, but no lidar2cam matrix compared with kitti dataset. Is lidar2cam matrix is necessary for training? If so, can I train with world2cam matrix?
Thank you!

**Comment by filaPro** [comment]
Yes, I don't see difference between lidar2img and world2img. We only compute projection matrix [here](https://github.com/saic-vul/imvoxelnet/blob/master/mmdet3d/models/detectors/imvoxelnet.py#L115) and use it [here](https://github.com/saic-vul/imvoxelnet/blob/master/mmdet3d/models/detectors/imvoxelnet.py#L145). Just check that your points from world coordinate system are projected to something reasonable in this function. Also be careful with world coordinate ranges for your custom dataset [1](https://github.com/saic-vul/imvoxelnet/blob/master/configs/imvoxelnet/imvoxelnet_kitti.py#L30), [2](https://github.com/saic-vul/imvoxelnet/blob/master/configs/imvoxelnet/imvoxelnet_kitti.py#L72).

Btw, we also implement ImVoxelNet in original mmdetection3d [repo](https://github.com/open-mmlab/mmdetection3d/tree/master/configs/imvoxelnet). The code may be more clean there.

**Comment by Italian-PAO** [comment]
Thank you so much for your reply!
Actually, I am using mmdet3d repo, and I have used the roation (without the translation) above to formulate "world2img" matrix, but when I browse my dataset in mmdet3d, the dir angle seems to have pi/4 offset, like this
![image](https://user-images.githubusercontent.com/49139695/199440690-01d67287-cf4b-4dbb-908f-8c93af8fb7bb.png)
I think maybe it's caused by "LiDARInstance3DBoxes" in mmdet3d, because when I browse the dataset with other model's config using "CameraInstance3DBoxes", the visualization result is right. 
I also tried to change the “[dir_offset](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/dense_heads/anchor3d_head.py#L60)” in anchor3d_head.py, but it still does not help.
What could be the reason of this problem？Or will this problem affect the detection results except the wrong angle.
Thanks again
